### PR TITLE
Loadout patch

### PR DESCRIPTION
**Pull request by HeroesMaker**
loadouts modification on PLA, JGSDF and ANZAC.
Added canadian army.
Added SMAW optics and spotting rounds to cargo selection.

### CODE REVIEW

File: loadouts/cargo-list.sqf [hunk 1]
```diff
@@ -127,6 +127,15 @@ switch (_x) do {
 			["R3F_FUM40",  40]
 		] call BRM_FMK_fnc_addItems;
 	};
+	
+	case "SMAW":{
```
Comment (Coryf88) [inline loadouts/cargo-list.sqf:131]: Instead add to "at", if specAT is a SMAW, so old missions will also get the items.

File: loadouts/factions/Canada.sqf [hunk 2]
```diff
@@ -0,0 +1,197 @@
+// INFO ========================================================================
+/*
+    A FUCKING LEAF
```
Comment (Coryf88) [inline loadouts/factions/Canada.sqf:3]: Unified military of Canada.

File: loadouts/factions/Canada.sqf [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    "BRM_LWH_MASK_"+_camo,
+    "BRM_LWH_MASK_COMS_"+_camo,
+    "BRM_LWH_GHILLIE_"+_camo,
+    "BRM_LWH_COMS_GHILLIE_"+_camo
+];
+
+_uniformsLIST = [
+    "BRM_BattleUNI_"+_camo,
+    "BRM_BattleUNIRS_"+_camo,
+    "BRM_BattleUNITS_"+_camo
```
Comment (Coryf88) [inline loadouts/factions/Canada.sqf:105]: `"BRM_BattleUNIT_"`

File: loadouts/factions/Canada.sqf [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+_mortar = "rhs_M252_Gun_Bag";
+_mortarTripod = "rhs_M252_Bipod_Bag";
+
+_UAVBag = "auto";
+_UAVTerminal = "auto";
+
+// VEHICLES ====================================================================
+
+    _factionVehicles = [
```
Comment (Coryf88) [inline loadouts/factions/Canada.sqf:167]: Unindent.

File: loadouts/factions/Canada.sqf [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    /*  Transport Trucks  */	,["C_TFC_VS_LSVW_transport_covered_F", "C_TFC_VS_LSVW_transport_uncovered_F"]
+    /*  Static Defence    */	,["RHS_Stinger_AA_pod_D", "RHS_M2StaticMG_D", "RHS_M2StaticMG_MiniTripod_D", "RHS_TOW_TriPod_D"]
+    /*  Boats             */	,["B_Boat_Transport_01_F", "B_Boat_Armed_01_minigun_F"]
+    /*  UAV               */	,["B_UAV_02_CAS_F","B_UAV_02_F"]
+    /*  UGV               */	,["B_UGV_01_F","B_UGV_01_rcws_F"]
+    /*  Support           */	,["rhsusf_M977A4_REPAIR_usarmy_wd","rhsusf_M977A4_AMMO_BKIT_usarmy_wd", "B_Truck_01_fuel_F"]
+    /*  Submarines        */	,["B_SDV_01_F"]
+    /*  MRAP Vehicles     */    ,["rhsusf_m113_usarmy_M2_90", "rhsusf_m113_usarmy_MK19", "rhsusf_M1117_D"]
+    ];
+};
```
Comment (Coryf88) [inline loadouts/factions/Canada.sqf:186]: Remove erroneous `};`

File: loadouts/factions/JSDF.sqf [hunk 6]
```diff
@@ -78,29 +78,36 @@ _countEpiCARGO = 30;
...
-    "TRYK_U_B_JSDF_CombatUniformTshirt"
-
-];
-
-_vestsLIST = [
-    "TRYK_V_PlateCarrier_JSDF"
-];
+    "BRM_BattleUNI_"+_camo,
+    "BRM_BattleUNIRS_"+_camo,
+    "BRM_BattleUNITS_"+_camo
```
Comment (Coryf88) [inline loadouts/factions/JSDF.sqf:105]: `"BRM_BattleUNIT_"`

File: loadouts/factions/anzac.sqf [hunk 7]
```diff
@@ -88,17 +88,19 @@ _countBloodbagCARGO = 20;
...
+    "BRM_ACH_COMS_"+_camo,
+    "BRM_ACH_MASK_COMS_"+_camo
 ];
 
 _uniformsLIST = [
-    "U_BRM_RS_DPCU",
-    "U_BRM_U_DPCU"
+    "BRM_BattleUNI_"+_camo,
+    "BRM_BattleUNIRS_"+_camo,
+    "BRM_BattleUNITS_"+_camo
```
Comment (Coryf88) [inline loadouts/factions/anzac.sqf:103]: `"BRM_BattleUNIT_"`

File: loadouts/factions/anzac.sqf [hunk 8]
```diff
@@ -132,12 +134,12 @@ _medicUNIFORM = _commonUNIFORM;
 _demoUNIFORM = _commonUNIFORM;
 _reconUNIFORM = _commonUNIFORM;
 
-_commonVEST    = "BRM_Protec_DPCU_RFL";
-_officerVEST   = "BRM_Protec_DPCU_SL";
-_ftlVEST       = "BRM_Protec_DPCU_TL";
+_commonVEST    = "BRM_Protec_"+_camo"_RFL";
```
Comment (Coryf88) [inline loadouts/factions/anzac.sqf:137]: Missing `+` after `_camo`.

File: loadouts/factions/anzac.sqf [hunk 9]
```diff
@@ -132,12 +134,12 @@ _medicUNIFORM = _commonUNIFORM;
...
+_officerVEST   = "BRM_Protec_"+_camo"_SL";
```
Comment (Coryf88) [inline loadouts/factions/anzac.sqf:138]: Missing `+` after `_camo`.

File: loadouts/factions/anzac.sqf [hunk 10]
```diff
@@ -132,12 +134,12 @@ _medicUNIFORM = _commonUNIFORM;
...
+_ftlVEST       = "BRM_Protec_"+_camo"_TL";
```
Comment (Coryf88) [inline loadouts/factions/anzac.sqf:139]: Missing `+` after `_camo`.

File: loadouts/factions/anzac.sqf [hunk 11]
```diff
@@ -132,12 +134,12 @@ _medicUNIFORM = _commonUNIFORM;
...
 _slVEST        = _officerVEST;
-_mgVEST        = "BRM_Protec_DPCU_MG";
-_grenadierVEST = "BRM_Protec_DPCU_GL";
+_mgVEST        = "BRM_Protec_"+_camo"_MG";
```
Comment (Coryf88) [inline loadouts/factions/anzac.sqf:141]: Missing `+` after `_camo`.

File: loadouts/factions/anzac.sqf [hunk 12]
```diff
@@ -132,12 +134,12 @@ _medicUNIFORM = _commonUNIFORM;
...
+_grenadierVEST = "BRM_Protec_"+_camo"_GL";
```
Comment (Coryf88) [inline loadouts/factions/anzac.sqf:142]: Missing `+` after `_camo`.

File: loadouts/faction-list.sqf [hunk 13]
```diff
@@ -20,6 +20,9 @@ switch (toUpper(_faction)) do {
     case "SLA": {
         #include "factions\sla.sqf"
     };
+    case "CANARMY": {
```
Comment (Coryf88) [inline loadouts/faction-list.sqf:23]: `CAF` instead?

**Review by Coryf88 — CHANGES_REQUESTED** [review]

**Review by HeroesMaker — COMMENTED** [review]
Made the changes requested, that'll teach me to do things half awake.

**Review by Coryf88 — APPROVED** [review]
👍

I tried out Github's `Allow edits by maintainers` feature.
Also, try to create separate PRs for well, separate changes. 😛

**Pull request merged**